solution.txt

```diff
@@ -60,7 +63,7 @@ Exercise 6:
 
 // Complete Exercise 6 below...
 
-foods.splice(foods.indexOf('pizza'),1,'sushi','cupcake')
+foods.splice(1, 1, 'sushi', 'cupcake');
```

Comment:
notice here how the second argument is 1, because you want to cut out one element at index value 1 (from the first argument).